.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Report a bug. Do not report security concerns this way.
+title: ''
+labels: bug
+assignees: ''
+---
+
+<!-- Please do not talk about security concerns here. Go to https://hackerone.com/github to report security concerns. -->
+
+## Describe the bug
+
+## Steps to reproduce the bug
```

Comment:
I do not think that this title is right for the stale action.
The bug template should be more in adequation with the fact that for a given configuration, there is an issue.
Providing the given stale config should not be required but at least suggested.
Also, referring to the debug section in the readme is important IMO